lib/screens/login/login.dart

```diff
@@ -0,0 +1,109 @@
+import 'package:ecellapp/screens/login/cubit/login_cubit.dart';
+import 'package:ecellapp/screens/login/widgets/email_field.dart';
+import 'package:ecellapp/screens/login/widgets/password_field.dart';
+import 'package:ecellapp/screens/signup/signup.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class LoginScreen extends StatelessWidget {
+  final TextEditingController passwordController = TextEditingController();
+  final TextEditingController emailController = TextEditingController();
+  final _formKey = GlobalKey<FormState>();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: BlocConsumer<LoginCubit, LoginState>(
+        listener: (context, state) async {
+          if (state is LoginSuccess) {
```

Comment:
We use the listener to show overlay UI (like snackbar/toast) or do other logic that is not directly handled by the builder. Since you are already showing a screen displaying "Login Success", you should not show a snackbar.